runtime-config.md

```diff
@@ -1,6 +1,12 @@
 ## Mount Configuration
 
-Additional filesystems can be declared as "mounts", specified in the *mounts* array. The parameters are similar to the ones in Linux mount system call. [http://linux.die.net/man/2/mount](http://linux.die.net/man/2/mount)
+Additional filesystems can be declared as "mounts", specified in the *mounts* object.
+Keys in this object are names of mount points from portable config.
+Values are objects with configuration of mount points.
+The parameters are similar to the ones in Linux mount system call.
+[http://linux.die.net/man/2/mount](http://linux.die.net/man/2/mount)
+Note, that only mount points which listed in portable configuration will be
+really mounted.
```

Comment:
I don't get "README's Markdown formatting" part. Why they should be on one line?